src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -2379,31 +2378,24 @@ namespace cryptonote
         KV_SERIALIZE_N(data.start_height, "start_height")
         KV_SERIALIZE(binary)
         KV_SERIALIZE(compress)
-        if (this_ref.binary)
+        if (this_ref.binary && this_ref.compress)
```

Comment:
Should the `compress` variable just be removed?

Reply:
Just from the `if`? I could do that.